backend/Cyberia/views.py

```diff
@@ -0,0 +1,102 @@
+from urllib import response
+from rest_framework.decorators import api_view
+from django.http import JsonResponse
+from dotenv import load_dotenv
+from rest_framework.response import Response
+from rest_framework import status
+from pathlib import Path
+import os
+import requests
+from Cyberia.NotionAPIparser import * 
+from threading import Thread
+
+dotenv_path = Path('backend/.env')
+load_dotenv(dotenv_path)
+token = os.getenv('token')
+version = os.getenv('version')
+components = dict()
```

Comment:
Can you please avoid making `components` a global variable, as it has potential to change the behavior when imported (https://google.github.io/styleguide/pyguide.html#25-global-variables). For `token` and `version`, it is still okay since it's just constants.

One solution can be to remove the `components` completely. And instead, you can return the data from each of your functions (overview(), benefit(), etc) instead of adding it to your global dictionary. So it would be something like:
```
data += overview(...)
data += benefit(...)
# and so on...
```

And thus in your functions, you can just do `return parse(data)` instead of `components[...] = parse(data)`

Reply:
Hi Kevin, thanks for your comment here. Actually, I used multithreading here, because fetching data for different parts sequentially could be very slow. Hence,  i don't think i can get the returned value like `data += overview(...)` ?